src/zip.js

```diff
@@ -10,7 +10,7 @@ module.exports = function(gj, options) {
 
     [geojson.point(gj), geojson.line(gj), geojson.polygon(gj)]
         .forEach(function(l) {
-        if (l.geometries.length) {
+        if (l.geometries[0].length) {
```

Comment:
Is `l.geometries` guaranteed to have at least one element? Could this check be `if (l.geometries.length && l.geometries[0].length)` to be more cautious?